src/main/java/duke/commands/ListCommand.java

```diff
@@ -0,0 +1,42 @@
+package duke.commands;
+
+import duke.storage.Storage;
+import duke.tasks.TaskList;
+import duke.ui.Ui;
+
+/**
+ * Responsible for listing the contents of TaskList.
+ */
+public class ListCommand extends Command {
+    /**
+     * Constructs a ListCommand object.
+     */
+    public ListCommand() {}
```

Comment:
Consider formatting this method using Egyptian style braces even though it has an empty body.